src/session_store.rs

```diff
@@ -59,9 +66,25 @@ where
         self.entries.get_mut(id).map(|e| e.unsubscribe(subscription_id));
     }
 
+    // Registers a callback for any change in the store.
+    // The callback is NOT called for the existing values.
+    // Only one callback can be registered at a time.
+    pub fn on_any_change(&mut self, f: impl Fn(&K, &V) + Send + 'static) {
+        self.on_any_change = Box::new(f);
```

Comment:
Maybe `assert!(self.on_any_change.is_none())` to make sure we are not accidentally overriding it?

Reply:
Done.